link/src/main/java/com/stripe/android/link/theme/Color.kt

```diff
@@ -28,22 +35,45 @@ private val DarkTextSecondary = Color(0x99EBEBF5)
 private val DarkTextDisabled = Color(0x61FFFFFF)
 private val DarkBackground = Color(0xFF1C1C1E)
 private val DarkFill = Color(0x33787880)
+private val DarkCloseButton = Color(0x99EBEBF5)
+private val DarkLinkLogo = Color.White
+private val DarkSecondaryButtonLabel = ActionGreen
 
 internal data class LinkColors(
     val componentBackground: Color,
     val componentBorder: Color,
     val componentDivider: Color,
     val buttonLabel: Color,
     val disabledText: Color,
+    val closeButton: Color,
+    val linkLogo: Color,
+    val secondaryButtonLabel: Color,
     val materialColors: Colors
 )
 
+@Composable
+internal fun PaymentsThemeForLink(
+    content: @Composable () -> Unit
+) {
+    val paymentsTheme = PaymentsThemeDefaults.colors(isSystemInDarkTheme())
```

Comment:
nit: maybe `paymentsThemeColors` since it's not the whole theme but just the colors?